romsel_aktheme/nitrofiles/language/lang_de/language.txt

```diff
@@ -9,64 +9,64 @@ Paste = Einfügen
 Delete = Löschen
 Setting = Einstellungen
 Info = Eigenschaften
-Hide = Verstecke
+Hide = Verstecken
 
 [message box]
 yes = Ja
 no = Nein
 ok = OK
-cancel = Abbr.
+cancel = Abbrechen
 
 [cheats]
 button = Cheats
 title = Cheats-Menü
 confirm = Bestätigen
-Cancel = Abbr.
+Cancel = Abbrechen
 info = Eigenschaften
-deselect = Deselect all
+deselect = Auswahl rückgangig machen
```

Comment:
Well, this strings too long for deselect button. And cannot be displayed well.

Reply:
I will update it. Thank you for letting me know!

Reply:
> Okay. Thanks for updating! I will try keeping translations to a minimum

Why not use 'Alle abwählen' for alt?

Reply:
Great idea lmao